src/util/emoji.py

```diff
@@ -0,0 +1,12 @@
+from typing import Union, TYPE_CHECKING
+import emoji
+from util.constants import CUSTOM_EMOJIS
+
+if TYPE_CHECKING:
+    from model.deepdives import Anomaly, MissionType, Warning
+
+def get_emoji(model: Union["MissionType", "Warning", "Anomaly", None]) -> Union[str, None]:
+    DEFAULT_EMOJI = emoji.emojize(':black_medium_square:');
+    if(model == None):
+        return DEFAULT_EMOJI
+    return CUSTOM_EMOJIS.get(model.value, DEFAULT_EMOJI)
```

Comment:
The `==` operator should pythonically be `is` when doing a check for `None`, but this can all be simplified to a one liner for readability:
```suggestion
    DEFAULT_EMOJI if model is None else CUSTOM_EMOJIS.get(model.value, DEFAULT_EMOJI)
```